components/Discussion/graphql/store.js

```diff
@@ -88,13 +88,11 @@ export const mergeComments = ({ parentId, appendAfter, comments }) => draft => {
 
     /*
      * Update the 'CommentConnection' of the parent comment.
-     *
-     * We clear 'nodes' because we don't need it. Though we shouldn't request it
-     * in the query at all, it's likely a mistake if we requested it.
      */
     draft.discussion.comments.nodes[parentIndex].comments = {
-      ...comments,
```

Comment:
`comments` is a `CommentConnection` which needs an `id` and `__typename`. IMO the old spread solution was IMO fine.

Reply:
I reverted the commit, and removed the comment above. GraphQL appears to do the right thing already.